unit-tests/live/calib/test_occ_calibrations.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (2017-2025), RealSense, Inc.
+# Certain Intel® RealSense™ products are sold by RealSense, Inc. under license from Intel Corporation),
+# This "Software" is furnished under license and may only be used or copied in accordance with the terms of that license.
+# No license, express or implied, by estoppel or otherwise, to any intellectual property rights is granted by this document.
+# The Software is subject to change without notice and should not be construed as a commitment by RealSense, Inc. or Intel Corporation to market, license, sell or support any product or technology.
+# Unless otherwise provided for in the license under which this Software is provided, the Software is provided AS IS, with no warranties of any kind, express or implied.
+# Except as expressly permitted by the Software license, neither RealSense, Inc. nor Intel Corporation, or any of their suppliers, assumes any responsibility or liability for any errors or inaccuracies that may appear herein.
+# Except as expressly permitted by the Software license, no part of the Software may be reproduced, stored in a retrieval system, transmitted in any form, or distributed by any means without the express written consent of RealSense, Inc.
+
+#########################################################################################################################################
+##                   tests for OCC and Tare calib flows                                                                ##
+#########################################################################################################################################
+import sys
+import time
+import pyrealsense2 as rs
+from rspy import test, log
+from test_calibrations_common import calibration_main
+
+#disabled until we stabilize lab
+#test:donotrun
+
+def on_chip_calibration_json(occ_json_file, host_assistance):
```

Comment:
host assistance actual value may be different when reading from json. Suggest to output it if reading file and pass to calibration_main call.